build/build/src/config.rs

```diff
@@ -7,8 +7,13 @@ use semver::VersionReq;
 
 
 
-pub fn load_yaml(yaml_text: &str) -> Result<Config> {
-    let raw = serde_yaml::from_str::<ConfigRaw>(yaml_text)?;
+/// Load the build configuration, based on the `build-config.yaml` and `.node-version` files in
+/// the repo root.
+pub fn load() -> Result<Config> {
+    let yaml_text = include_str!("../../../build-config.yaml");
+    let node_version = include_str!("../../../.node-version").trim();
```

Comment:
Please load these files at runtime. Some team members want to build and install the build script globally and then reuse it. Well, it gets broken occasionally anyway, but let's not make it worse.

Reply:
But reading those files in runtime sounds like a good idea anyway.